packages/nimble-components/src/rich-text/components/mention-view/mention-users-view/styles.ts

```diff
@@ -0,0 +1,37 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+    mentionFont,
+    mentionFontColor,
+    mentionDisabledFontColor,
+    bodyEmphasizedFontColor
+} from '../../../../theme-provider/design-tokens';
+
+export const styles = css`
+    ${display('inline-block')}
+
+    :host {
+        box-sizing: border-box;
+        font: ${mentionFont};
+        color: ${bodyEmphasizedFontColor}
+        white-space: normal;
+        --ni-nimble-icon-color: ${mentionFontColor};
+    }
+
+    :host(:not([disable-editing])) .control {
```

Comment:
Avoid a bunch of `:not` selectors and rely on the cascade: https://github.com/ni/nimble/blob/main/packages/nimble-components/docs/css-guidelines.md#prefer-cascade-for-state-changes

```
.control {
   display: none;
}

:host([disable-editing]) {
   display: whatever it is
}
```

Reply:
Thanks for sharing the doc. I have updated the style as suggested.